feature/home/src/main/java/com/terning/feature/home/component/bottomsheet/JobFilteringScreen.kt

```diff
@@ -0,0 +1,102 @@
+package com.terning.feature.home.component.bottomsheet
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import com.terning.core.designsystem.extension.noRippleClickable
+import com.terning.core.designsystem.theme.Grey200
+import com.terning.core.designsystem.theme.Grey300
+import com.terning.core.designsystem.theme.Grey350
+import com.terning.core.designsystem.theme.TerningMain
+import com.terning.core.designsystem.theme.TerningTheme
+import com.terning.core.designsystem.type.JobType
+
+@Composable
+fun JobFilteringScreen(
+    initOption: Int,
+    onButtonClick: (JobType) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var selectedIndex by remember { mutableIntStateOf(initOption) }
+
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(3),
+        horizontalArrangement = Arrangement.spacedBy(10.dp),
+        verticalArrangement = Arrangement.spacedBy(12.dp),
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(bottom = 60.dp, start = 23.dp, end = 23.dp),
+
+        ) {
```

Comment:
파라미터가 3개 이상 넘어가면 줄바꿈을 해주는 게 가독성에 좋을 것 같아요....!!
```suggestion
        modifier = modifier
            .fillMaxWidth()
            .wrapContentHeight()
            .padding(
                bottom = 60.dp, 
                start = 23.dp, 
                end = 23.dp),
        ) {
```